src/assets/scss/components/spinner.scss

```diff
@@ -0,0 +1,5 @@
+.alg-spinner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
Why is this code located in global scss assets instead of the component scss ?

Reply:
Fixed